### ISSUE TITLE
Do you use validation dataset?

### ISSUE THREAD

**Issue by qianlanwyd** [Megvii-BaseDetection/DisAlign#14]
https://github.com/Megvii-BaseDetection/DisAlign/blob/main/classification/imagenetlt/resnext50/resx50.scratch.imagenet_lt.224size.90e.disalign.10e/config.py#L31

It seems that you only use test dataset? What is the reason for that?

**Comment by tonysy** [comment]
We use the validation set for hyper-parameter tuning and use the test set to report final result.
